src/otx/algo/classification/torchvision_model.py

```diff
@@ -139,6 +139,13 @@ def __init__(
         self.softmax = nn.Softmax(dim=-1)
         self.loss = loss
 
+        avgpool_index = 0
+        for i, layer in enumerate(self.backbone.children()):
+            if layer.__class__.__name__ == "AdaptiveAvgPool2d":
```

Comment:
Thank you for this suggestion, updated accordingly.
Yes, I checked that it works on all torchvision models that we have:
![image](https://github.com/openvinotoolkit/training_extensions/assets/33455576/a6a66e92-24a4-4ff4-81bb-09fab444a795)

Only `regnet_y_128gf` model fails because of unavailable weights on the server